.prettierignore

```diff
@@ -0,0 +1 @@
+src/components/tiptap/notificationWidget/NotificationWidget.tsx
```

Comment:
why are we ignoring this file?

Reply:
It was a quick hack for before since prettier was throwing indentation errors for ternary operators used in this file. Will check the issue again and remove the `.prettierignore` file. @saugatmaharjan